### PR TITLE
Add Documentation R4 Organization Writes

### PR DESCRIPTION
**Pull request by sbanda51**
Added Documentation for R4 Organization Writes

### CODE REVIEW

File: content/millennium/r4/entities/organization.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on a subsequent update.
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
```
Comment (ebaranowski) [inline content/millennium/r4/entities/organization.md]: nit: blank lines

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 2]
```diff
@@ -61,6 +61,17 @@ The Encounter Resource supports proprietary codes for:
...
+
+<!-- use html header to avoid showing up in toc -->
+<h4>Organization</h4>
+
+The Organization Resource supports proprietary codes for:
+
+* Organization.identifier.type codes are maintained in [Code Set 334 Organization Alias Type](#code-set-334-organization-alias-type)
+* Organization.type codes are maintained in [Code Set 278 Organization Type](#code-set-278-organization-type)
+
+
```
Comment (ebaranowski) [inline content/millennium/r4/proprietary-codes-and-systems.md]: nit: blank line

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 3]
```diff
@@ -287,6 +298,17 @@ This code set maintains medication administration sites such as left arm, right
       "display": "Right arm", 
       "userSelected": true 
     }
+    
+##### Code Set 278 Organization Type
```
Comment (ebaranowski) [inline content/millennium/r4/proprietary-codes-and-systems.md]: same comments from read

File: lib/resources/example_json/r4_examples_organization.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+module Cerner
+  module Resources
+ R4_ORGANIZATION_CREATE ||= {
```
Comment (ebaranowski) [inline lib/resources/example_json/r4_examples_organization.rb]: spacing is off

File: lib/resources/r4/allergy_intolerance.yaml [hunk 5]
```diff
@@ -77,7 +77,7 @@ fields:
     description: The clinical status of the allergy or intolerance.
     terminology:
     - display: AllergyIntolerance Clinical Status Codes
-      system: http://terminology.hl7.org/CodeSystem/allergyintolerance-clinical
+      system: http://terminology.hl7.org/CodeSystem/aller gyintolerance-clinical
```
Comment (ebaranowski) [inline lib/resources/r4/allergy_intolerance.yaml]: ??

File: lib/resources/r4/organization.yaml [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    description: The type of the FHIR resource.
+    action: create
+    example: |
+      {
+        "resourceType": "Organization"
+      }
+    note: resourceType must be Organization.
+    url: https://hl7.org/fhir/R4/json.html#resources
+
+  - name: identifier
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: So these should mix into the read ones. Shouldnt need 2

File: lib/resources/r4/organization.yaml [hunk 7]
```diff
@@ -0,0 +1,141 @@
+---
+name: Organization
+field_name_base_url: https://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: needs the updates from Read

File: lib/resources/r4/organization.yaml [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+    url: https://hl7.org/fhir/R4/json.html#resources
+
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: The identifiers for the Organization.
+    action: create
+    example: |
+      {
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: spacing is off

File: lib/resources/r4/organization.yaml [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+                                "code": "NPI"
+                            }
+                        ]
+                    }
+                }
+            ]
+      }
+    note: |
+      <ul>
+        <li> If set, The provided identifier must meet the following conditions:
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: lowercase The

File: lib/resources/r4/organization.yaml [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.name
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: OrganizationType
+    description: The Organization Type.
+    action: create
+    example: |
+      {
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: spacing is off

File: lib/resources/r4/organization.yaml [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+        "active": true
+      }
+    note: If set, value must be true.
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.active
+
+  - name: name
+    required: 'Yes'
+    cardinality: 0..1
+    type: OrganizationName
+    description: The names associated with the individual.
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: Name used for the organization.

File: lib/resources/r4/organization.yaml [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+      <ul>
+        <li> At least one name <b>must</b> be provided. </li>
+      </ul>
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.name
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: OrganizationType
+    description: The Organization Type.
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: Kind of organization.

File: lib/resources/r4/organization.yaml [hunk 13]
```diff
@@ -0,0 +1,141 @@
...
+    note: |
+      <ul>
+        <li> At least one name <b>must</b> be provided. </li>
+      </ul>
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.name
+
+  - name: type
+    required: 'No'
+    cardinality: 0..*
+    type: OrganizationType
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: CodeableConcept

File: content/millennium/r4/entities/organization.md (hunk 1 above)
Comment (sbanda51) [reply content/millennium/r4/entities/organization.md]: fixed here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 2 above)
Comment (sbanda51) [reply content/millennium/r4/proprietary-codes-and-systems.md]: fixed here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 3 above)
Comment (sbanda51) [reply content/millennium/r4/proprietary-codes-and-systems.md]: Added here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-6fdcb00b5b1b6edd51b687794bea6b95R303

File: lib/resources/r4/allergy_intolerance.yaml (hunk 5 above)
Comment (sbanda51) [reply lib/resources/r4/allergy_intolerance.yaml]: my bad fixed here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-eeda3e608dc0db5d6d7c877900383353R80

File: lib/resources/r4/organization.yaml (hunk 6 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R9

File: lib/resources/r4/organization.yaml (hunk 7 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R9

File: lib/resources/r4/organization.yaml (hunk 8 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: fixed here https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R9

File: lib/resources/r4/organization.yaml (hunk 9 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R31

File: lib/resources/r4/organization.yaml (hunk 10 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R31

File: lib/resources/r4/organization.yaml (hunk 11 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R117

File: lib/resources/r4/organization.yaml (hunk 12 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R60

File: lib/resources/r4/organization.yaml (hunk 13 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: https://github.com/cerner/fhir.cerner.com/pull/326/commits/6f310109291d4e63675659da7d48656d75534c9e#diff-b08823bfbc91ea717475d1d2372baf64R59

File: lib/resources/r4/organization.yaml (hunk 8 above)
Comment (ebaranowski) [reply lib/resources/r4/organization.yaml]: Im pretty sure the spacing is still off

File: lib/resources/r4/organization.yaml (hunk 10 above)
Comment (ebaranowski) [reply lib/resources/r4/organization.yaml]: Pretty sure the spacing is still off

File: lib/resources/r4/organization.yaml (hunk 8 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: sorry fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/f2f4341b4d5d18c180e40f44cd8daa2c3e3ed8fd

File: lib/resources/r4/organization.yaml (hunk 10 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: fixed https://github.com/cerner/fhir.cerner.com/pull/326/commits/f2f4341b4d5d18c180e40f44cd8daa2c3e3ed8fd

File: lib/resources/r4/organization.yaml (hunk 8 above)
Comment (ebaranowski) [reply lib/resources/r4/organization.yaml]: Fixed: https://github.com/cerner/fhir.cerner.com/pull/326/commits/fb4db54a3c8e0b86d489a4e434f5277a56342255#diff-b08823bfbc91ea717475d1d2372baf64R15

File: lib/resources/r4/organization.yaml (hunk 10 above)
Comment (ebaranowski) [reply lib/resources/r4/organization.yaml]: Fixed: https://github.com/cerner/fhir.cerner.com/pull/326/commits/fb4db54a3c8e0b86d489a4e434f5277a56342255#diff-b08823bfbc91ea717475d1d2372baf64R64

File: lib/resources/r4/organization.yaml [hunk 14]
```diff
@@ -0,0 +1,127 @@
+---
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: Take a look at FinancialTransactions URLs and then yours. You should update yours accordingly :)

File: lib/resources/r4/organization.yaml [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+name: Organization
+field_name_base_url: http://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: Identifies this organization across multiple systems.
+    url: http://hl7.org/fhir/R4/datatypes.html#Identifier
```
Comment (nishanthnamana) [inline lib/resources/r4/organization.yaml]: Same here. All links should be secured ones.

File: lib/resources/r4/organization.yaml [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+        <li> If set, the provided identifier must meet the following conditions:
+          <ul>
+            <li> Must contain the `system`, `value`, and `type` fields.</li>
+            <li> Must contain only an NPI, OID or TAXID system. </li>
+          </ul>
+        </li>
+      </ul>
+    children:
+
+      - name: type
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: This rendered in creates. I misled you. We need Identifier for creates, and then this is reads. My bad

File: lib/resources/r4/organization.yaml (hunk 15 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated https://github.com/cerner/fhir.cerner.com/pull/326/commits/7e58be985de3b799900d13d575b7699ab10b9460

File: lib/resources/r4/organization.yaml [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+      {
+        "active": true
+      }
+    note: If set, value must be true.
+    url: https://hl7.org/fhir/R4/organization-definitions.html#Organization.active
+
+  - name: name
+    required: 'Yes'
+    cardinality: 0..1
+    type: OrganizationName
```
Comment (ebaranowski) [inline lib/resources/r4/organization.yaml]: this is a string

File: content/millennium/r4/entities/organization.md [hunk 18]
```diff
@@ -0,0 +1,83 @@
...
+X-Runtime: 2.011826
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on a subsequent update.
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
```
Comment (ebaranowski) [inline content/millennium/r4/entities/organization.md]: Missing the errors and OperationOutcome links

**Comment by ebaranowski** [comment]
Validation:

Screen Shot 2019-12-03 at 5 16 46 PM

File: lib/resources/r4/organization.yaml (hunk 14 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3

File: lib/resources/r4/organization.yaml (hunk 16 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3

File: lib/resources/r4/organization.yaml (hunk 17 above)
Comment (sbanda51) [reply lib/resources/r4/organization.yaml]: updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3

File: content/millennium/r4/entities/organization.md (hunk 18 above)
Comment (sbanda51) [reply content/millennium/r4/entities/organization.md]: added here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3

**Comment by nishanthnamana** [comment]
fhir.cerner.com Validation
Organization_1

Organization_4

**Pull request merged**